docs/cache.md

```diff
@@ -0,0 +1,26 @@
+# Triage Metadata Caching
+
+The [wpt.fyi](https://wpt.fyi) dashboard has support for linking test results
+for a specific test and browser to issues or bugs (or more generically, any
+URL). The triaged data is stored in the
+[wpt-metadata](https://github.com/web-platform-tests/wpt-metadata) repository
+and is reflected back onto [wpt.fyi](https://wpt.fyi).
+
+This section explains the caching mechanisms for triage data in webapp and
+searchcache.
+
+## webapp
+Webapp hosts the following API endpoints:
+
+- `/api/metadata` returns WPT Metadata to be displayed on the bottom of wpt.fyi pages and via inline icons on the test-result table
+
+- `/api/metadata/triage` records triage information in the backend and sends PRs to the wpt-metadata repository which acts as the backing data store.
+
+Both endpoints share the same in-memory copy of the wpt-metadata repository, and this copy expires [every 10 minutes](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/metadata_cache.go#L29) in Redis. The only exception is that when users triage metadata, /api/metadata/triage will force an update of the copy.
```

Comment:
It seems likely that this link will become outdated (i.e. a specific line number in the current version of the file).  Maybe this should link to a [specific commit](https://docs.github.com/en/github/managing-your-work-on-github/creating-a-permanent-link-to-a-code-snippet)? Or name the function here, so the reader can find it?

This applies to other code links in the doc.

Reply:
I will use permalinks